Mathlib/Logic/Equiv/TransferInstance.lean

```diff
@@ -135,7 +135,7 @@ noncomputable instance [Small.{v} α] (R : Type*) [SMul R α] : SMul R (Shrink.{
   (equivShrink α).symm.smul R
 
 /-- Transfer `Pow` across an `Equiv` -/
-@[to_additive (attr := reducible) existing smul]
+@[reducible, to_additive existing smul]
```

Comment:
This was adding the reducible attribute a second time, which will become illegal in v4.9.0 (as a sanity check).

Reply:
I don't think either the old or the new spelling works correctly:
```
import Mathlib

@[to_additive (attr := reducible)]
def one := 1

example : one = 1 := by with_reducible rfl
example : zero = 0 := by with_reducible rfl


@[reducible, to_additive]
def one' := 1

example : one' = 1 := by with_reducible rfl
example : zero' = 0 := by with_reducible rfl
```

Reply:
I think the correct fix is `attribute [reducible] Equiv.smul` on the line below, maybe with a TODO about `to_additive`.

Reply:
Note that `smul` is already defined several lines above, and has `reducible` there already.